src/js/components/inspectors/MultiValueProperty.test.jsx

```diff
@@ -0,0 +1,239 @@
+// /* eslint no-unused-expressions:0,new-cap:0 */
```

Comment:
i just realized that i misunderstood your question earlier. i thought you were asking about old tests, but i see that you wrote new tests!

i've never run the tests before but i think that you can try to run these using `yarn test`. see `package.json` for a list of scripts that you can run with yarn

Reply:
so i didn't actually write this. i just copied them from property so they are actually wrong. :/ so will probably leave it commented out.

Reply:
wdyt?

Reply:
ah ok, that's fine to just leave it then! all the tests haven't been updated since before the refactor, and since we're actively making changes at a fast pace, it's not worth investing much time into updating the tests until things are a bit more stable IMO